tests/suites/static_analysis/lint_go.sh

```diff
@@ -38,6 +38,20 @@ run_domain_imports() {
 	done
 }
 
+run_juju_errors_imports() {
+    pkgs=("domain")
+
+    for pkg in "${pkgs[@]}"; do
+        dirs=$(find ${pkg} -mindepth 1 -maxdepth 10 -type d | sort -u)
+        for subdir in $dirs; do
+            echo "$subdir"
+            imports=$(go list -json -e -test "./${subdir}" 2>/dev/null | jq -r ".Imports // [] | .[]")
```

Comment:
Any reason why we didn't stick to using `got=$(go run ./scripts/import-inspector ...)` like elsewhere?

Reply:
because the import inspector looks at dependencies. We just want to look at direct imports.